src/sentry/tagstore/snuba/backend.py

```diff
@@ -589,10 +590,17 @@ def get_tag_value_paginator_for_projects(
         if snuba_key in BLACKLISTED_COLUMNS:
             snuba_key = "tags[%s]" % (key,)
 
-        if query:
-            conditions.append([snuba_key, "LIKE", u"%{}%".format(query)])
+        if key in SearchVisitor.numeric_keys:
+            if query:
+                conditions.append([snuba_key, ">=", int(query) - 50])
```

Comment:
You might want to guard against the ValueError here as someone will put `3bears` into the numeric key value and we don't want to 500 on that.

Reply:
:+1: Added an `isdigit` check so we default to the all values condition instead.